test/algo/nas/test_cgo_engine.py

```diff
@@ -162,7 +162,7 @@ def _new_trainer():
 
 
 def _load_mnist(n_models: int = 1):
-    path = Path(__file__).parent / 'mnist_pytorch.json'
+    path = Path('ut/nas/mnist_pytorch.json')
```

Comment:
I suggest to keep `__file__`. It is dangerous to assert pwd.

Reply:
It should be ``pytestconfig.rootpath / ...``. But the fixture doesn't work well with python unittest module. And it's a CGO engine test -- I don't want to get my hands dirty.